py3status/modules/docker.py

```diff
@@ -0,0 +1,89 @@
+r"""
+Display properties of running docker containers.
+
+Configuration parameters:
+    containers: filter the list by container or image name (default [])
+    delimiter: separator between containers (default ' ')
+    format: format to display each container (default '{name} {image}')
+
+Format placeholders:
+    {name} instance name (--name= on run, or randomly generated)
+    {short_id} shortened unique container id
+    {id} full container id
+    {status} running state of container
+    {image} name of the image running in the container
+    {gateway} the gateway IP to interact with the container
+
+Requires:
+    docker: python client for the docker container engine
+
+Examples:
+```
+docker {
+    containers = ["mysql:*"]
+    delimiter = "|"
+    format = "{name}({image})- {gateway}"
+}
+```
+
+@author la kevna (Aaron Moore)
+
+SAMPLE OUTPUT
+{'full_text': u'vigorous_ride mysql:latest'}
+"""
+
+import docker
+from fnmatch import fnmatch
+
+
+class Py3status:
+    """
+    """
+
+    # available configuration parameters
+    containers = []
+    delimiter = " "
+    format = "{name} {image}"
```

Comment:
- It should be changed to `format = "{format_container}"` where a list of containers will be given to `format`.
- Then a new config `format_container = "{name} {image}"` for the containers.
- Rename `delimiter` to `format_container_separator` to be used with config `format_container`. 
- - Take a look at module `nvidia_smi` or any modules containing config `format_*_separator` for some examples on looping / merging.